os/binfmt/binfmt_exec.c

```diff
@@ -152,7 +152,9 @@ int exec(FAR const char *filename, FAR char *const *argv, FAR const struct symta
 	uint32_t size = 0;
 	struct tcb_s *tcb;
 
-	if (mm_allocate_ram_partition(&start_addr, &size) < 0) {
+	start_addr = kmm_memalign(size, size);
```

Comment:
There is no default value of size variable. So it always returns NULL.

Reply:
Yes. I am aware that it will always be NULL. The problem is that we dont have any way to find out the required size in this case. Also, according to our current requirements, we never use exec api to run an app. So, I propose that we disable the complete exec API and related implementation until we finalize its requirement and design. 
@sunghan-chang Please let me know your opinion on this.

Reply:
@jeongchanKim 
Regarding your second query, its true that the app's heap will be inside the app's partition which will now be allocated inside the kernel heap.

Reply:
I'm not sure what was my second query.
Anyway is it okay app's heap is inside the kernel? not user heap?

Reply:
Right. It is ok. The app heap is inside the app's partition which is allocated from the kernel. The access to this partition is controlled using the MPU regions.